package.json

```diff
@@ -20,8 +24,9 @@
     "lint:ts": "vale docs-src/typescript/*.md",
     "serve": "docusaurus serve",
     "start": "docusaurus start",
+    "snipsync": "snipsync",
     "swizzle": "docusaurus swizzle",
-    "worker": "pushd assembly && yarn worker && popd",
+    "worker": "cd assembly && yarn worker && cd ..",
```

Comment:
Getting this error on cloud because it is running a local worker and not a connection to cloud.

```
Workflow bundle created { size: '0.68MB' }
TransportError: tonic::transport::Error(Transport, hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 61, kind: ConnectionRefused, message: "Connection refused" })))
    at InternalNativeConnection.connect (/Users/rachfop/documentation/assembly/node_modules/@temporalio/worker/lib/connection.js:55:23)
    at async Worker.create (/Users/rachfop/documentation/assembly/node_modules/@temporalio/worker/lib/worker.js:187:51)
    at async useLocal (file:///Users/rachfop/documentation/assembly/worker.js:42:20)
    at async run (file:///Users/rachfop/documentation/assembly/worker.js:14:7)
[nodemon] app crashed - waiting for file changes before starting...
```